src/algorithms/reco/TransformBreitFrame.h

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 John Lajoie
+
+#pragma once
+
+#include <edm4hep/MCParticleCollection.h>
+#include <edm4eic/InclusiveKinematicsCollection.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <spdlog/logger.h>
+#include <map>
+#include <memory>
+#include <string>
+
+namespace eicrecon {
+
+  class TransformBreitFrame {
+
+    public:
+
+      // algorithm initialization
+      void init(std::shared_ptr<spdlog::logger> logger);
+
+      // run algorithm
+      std::unique_ptr<edm4eic::ReconstructedParticleCollection> process(const edm4hep::MCParticleCollection *mcpart,
+                                                                        const edm4eic::InclusiveKinematicsCollection *kine,
+                                                                        const edm4eic::ReconstructedParticleCollection *lab_collection);
```

Comment:
Then this would become:

```suggestion
      void process(const Input&, const Output&) const final;
```